### PR TITLE
`Table` bug bash fixes 

### PR DESCRIPTION
**Pull request by shaneeza**
## ✍️ Proposed changes

🎟 _Jira ticket:_ [Name of ticket](https://jira.mongodb.org/browse/[name-of-ticket])

Fixes bugs listed in this doc: https://docs.google.com/spreadsheets/d/1_YkJ5f8KU5wVN-v-UizYzNeAaCPdUSKLXefw3fzVx-s/edit?gid=0#gid=0. Scroll to the right of the doc to see my updated comments.

### For bug fixes, new features & breaking changes

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have run `yarn changeset` and documented my changes

### For new components

- [ ] I have added my new package to the global tsconfig
- [ ] I have added my new package to the Table of Contents on the global README
- [ ] I have verified the Live Example looks as intended on the design website.

### CODE REVIEW

File: packages/table/src/Table.Interactions.stories.tsx [hunk 1]
```diff
@@ -33,10 +33,6 @@ import {
 const meta: StoryMetaType<typeof Table> = {
   title: 'Components/Table/Interactions',
   component: Table,
-  // argTypes: {
-  //   shouldAlternateRowColor: { control: 'boolean' },
-  //   darkMode: { control: 'boolean' },
-  // },
```
Comment (tsck) [inline packages/table/src/Table.Interactions.stories.tsx:39]: Should we just remove this?

File: packages/table/src/Table/TableWithVS.stories.tsx [hunk 2]
```diff
@@ -642,6 +697,8 @@ export const WithLeafyGreenComponents: StoryFn<StoryTableProps> = args => {
       {
         accessorKey: 'clusterType',
         header: 'Cluster Type',
+        // his makes the size auto
```
Comment (tsck) [inline packages/table/src/Table/TableWithVS.stories.tsx:700]: Nit:
```suggestion
        // This makes the size auto
```

**Review by tsck — COMMENTED** [review]
Couple of tiny things but mostly LGTM

File: packages/table/src/Table.Interactions.stories.tsx (hunk 1 above)
Comment (shaneeza) [reply packages/table/src/Table.Interactions.stories.tsx:39]: yeah, we can remove it.

Comment (tsck) [reply packages/table/src/Table.Interactions.stories.tsx:39]: Ngl, it was early in the morning and I thought this was added in 😅. Ignore me lol

**Review by tsck — APPROVED** [review]

**Pull request merged**